plugins/ocp/meson.build

```diff
@@ -6,5 +6,9 @@ sources += [
   'plugins/ocp/ocp-fw-activation-history.c',
   'plugins/ocp/ocp-telemetry-decode.c',
   'plugins/ocp/ocp-hardware-component-log.c',
+  'plugins/ocp/ocp-print.c',
+  'plugins/ocp/ocp-print-stdout.c',
+  'plugins/ocp/ocp-print-json.c',
```

Comment:
The json file should only be added when the json library is available. And then you could update the `plugins/meson.build` file to always build the ocp plugin.

The rest looks good!

Reply:
Update the meson.build files by the additional commit. Thank you.
  Note: Still some exisitng ocp commands files needed to split json code.

Reply:
So do you want me to wait with merging this and wait until you finished the json code splitting?

Reply:
No need to wait for me. Still working on this but I am okay to do it by additional PRs. Thank you.